js/searchbar/customBangs.js

```diff
@@ -376,3 +377,17 @@ bangsPlugin.registerCustomBang({
     }
   }
 })
+
+    const data = settings.get('customBangs')
+    if(data) {
```

Comment:
This should be linted :)

Reply:
I'll do that once I've fixed everything else